src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/EncoderManager.h

```diff
@@ -1,3 +1,28 @@
+/*
+ * Copyright (c) 2019, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do we include starting year for files which were added in JDK projects like Lanai when we send out common review for OpenJDK? Is this a common practice?